packages/documentation/cypress/snapshots/utilities/text-truncation.snapshot.ts

```diff
@@ -0,0 +1,7 @@
+describe('Text Truncation', () => {
+  it('text truncation', () => {
+    cy.visit('/iframe.html?id=snapshots--text-truncation');
+    cy.get('.text-truncation-example', { timeout: 30000 }).should('be.visible');
+    cy.percySnapshot('Text Truncation', { widths: [320, 1440] });
```

Comment:
The class is not responsibve, therefore it is not necessary to test it on mobile.
```suggestion
    cy.percySnapshot('Text Truncation', { widths: [1440] });
```